testutils/helper/install.go

```diff
@@ -170,6 +171,32 @@ func (h *SoloTestHelper) InstallGloo(ctx context.Context, deploymentType string,
 	return nil
 }
 
+// Wait for the glooctl install command to respond, err on timeout.
+// The command returns as soon as the certgen pod starts, which should
```

Comment:
technically it returns after certgen completes and all other deployments have been applied

Reply:
Updated!